python/ql/src/Security/CWE-312/CleartextLogging.ql

```diff
@@ -0,0 +1,41 @@
+/**
+ * @name Clear-text logging of sensitive information
+ * @description Logging sensitive information without encryption or hashing can
+ *              expose it to an attacker.
+ * @kind path-problem
+ * @problem.severity error
+ * @precision high
+ * @id py/clear-text-storage-sensitive-data
```

Comment:
This still seems to be wrong. Suggest that it should match the `id` for JavaScript:
```suggestion
 * @id py/clear-text-logging
```